installer/opensuse/defaults

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file is sourced from main.sh to update distro-specific defaults.
+# It must set at least ARCH and MIRROR if not already specified.
+
+if [ -z "$ARCH" ]; then
+    ARCH="$(uname -m)"
+fi
+
+case "$ARCH" in
+arm*) ARCH="armv7hl";;
+arm64 | aarch64) ARCH="arm64";;
+*) error 2 "Invalid architecture '$ARCH'.";;
```

Comment:
wait what happened to the x86 architectures

Reply:
I dropped it for now, as there does no JeOS image exist, but just a installer image. That's something for the future. Creating an x86 image in the open build service and use it.

Reply:
definitely need x86 support before we can merge; this would confuse most of our users